Create User entity

kurojiinn opened the pull request:
Создание User сущности

kurojiinn committed:
Создал сущность User

fifimova commented on src/main/java/ru/kaInc/shelterbot/model/User.java:
```diff
@@ -0,0 +1,58 @@
+    }
+
+    @Override
+    public boolean equals(Object o) {
```
для энтити лучше переопределять equals и hashCode только по id

fifimova commented on src/main/java/ru/kaInc/shelterbot/model/User.java:
```diff
@@ -0,0 +1,58 @@
+    private String phone;
+
+    @ManyToOne()
+    @JoinColumn(name = "shelterId")
```
колонка будет shelter_id

fifimova commented on src/main/java/ru/kaInc/shelterbot/model/User.java:
```diff
@@ -0,0 +1,58 @@
+    private Shelter shelterId;
+
+    @ManyToOne()
+    @JoinColumn(name = "volunteerId")
```
volunteer_id

fifimova reviewed (approved):
аппрув!

fifimova commented on src/main/java/ru/kaInc/shelterbot/model/User.java:
```diff
@@ -0,0 +1,58 @@
+    private Volunteer volunteerId;
+
+    @Column(name = "isAdopter")
+    private Boolean isAdopter;
```
is_adopter

kurojiinn committed:
Изменил методы equals and hashCode теперь только id.
Изменил название колонок с
shelterId -> shelter_id,
volunteerId -> volunteer_id,
isAdopter -> is_adopter

HellHorseman commented:
Годно!!!

HellHorseman commented:
Годно

HellHorseman commented:
годно
